cli/cert.go

```diff
@@ -94,7 +94,9 @@ The certs directory should contain a CA cert and key.
 // to their corresponding files.
 func runCreateClientCert(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
-		cmd.Usage()
+		if err := cmd.Usage(); err != nil {
```

Comment:
This pattern is common here. Perhaps add a `mustUsage(cmd)` function to this package.